Description: Set animation defaults
Author: ?

--- a/metadata/animation.xml.in
+++ b/metadata/animation.xml.in
@@ -37,8 +37,8 @@
 	    <extensible/>
 	    <sort start="2"/>
 	    <default>
-	      <value>animation:Zoom</value>
-	      <value>animation:Fade</value>
+	      <value>animation:Glide 2</value>
+	      <value>animation:None</value>
 	      <value>animation:Fade</value>
 	    </default>
 	  </option>
@@ -49,9 +49,9 @@
 	    <min>50</min>
 	    <max>4000</max>
 	    <default>
-	      <value>200</value>
-	      <value>150</value>
-	      <value>150</value>
+	      <value>80</value>
+	      <value>80</value>
+	      <value>80</value>
 	    </default>
 	  </option>
 	  <option name="open_matches" type="list">
@@ -59,8 +59,8 @@
 	    <_long>The windows that will be animated.</_long>
 	    <type>match</type>
 	    <default>
-	      <value>(type=Normal | Dialog | ModalDialog | Unknown) &amp; !(name=mate-screensaver)</value>
-	      <value>(type=Menu | PopupMenu | DropdownMenu)</value>
+ 	      <value>((type=Normal | Unknown) | name=sun-awt-X11-XFramePeer | name=sun-awt-X11-XDialogPeer) &amp; !(role=toolTipTip | role=qtooltip_label) &amp; !(type=Normal &amp; override_redirect=1) &amp; !(name=mate-screensaver)</value>
+ 	      <value>(type=Menu | PopupMenu | DropdownMenu | Dialog | ModalDialog | Normal)</value>
 	      <value>(type=Tooltip | Notification | Utility) &amp; !(name=compiz) &amp; !(title=notify-osd)</value>
 	    </default>
 	  </option>
@@ -103,9 +103,9 @@
 	    <extensible/>
 	    <sort start="2"/>
 	    <default>
-	      <value>animation:Zoom</value>
 	      <value>animation:Fade</value>
 	      <value>animation:Fade</value>
+	      <value>animation:None</value>
 	    </default>
 	  </option>
 	  <option name="close_durations" type="list">
@@ -115,9 +115,9 @@
 	    <min>50</min>
 	    <max>4000</max>
 	    <default>
-	      <value>200</value>
-	      <value>150</value>
-	      <value>150</value>
+	      <value>80</value>
+	      <value>80</value>
+	      <value>50</value>
 	    </default>
 	  </option>
 	  <option name="close_matches" type="list">
@@ -125,8 +125,8 @@
 	    <_long>The windows that will be animated.</_long>
 	    <type>match</type>
 	    <default>
-	      <value>(type=Normal | Dialog | ModalDialog | Unknown) &amp; !(name=mate-screensaver)</value>
-	      <value>(type=Menu | PopupMenu | DropdownMenu)</value>
+ 	      <value>((type=Normal | Unknown) | name=sun-awt-X11-XFramePeer | name=sun-awt-X11-XDialogPeer) &amp; !(role=toolTipTip | role=qtooltip_label) &amp; !(type=Normal &amp; override_redirect=1) &amp; !(name=mate-screensaver)</value>
+ 	      <value>(type=Menu | PopupMenu | DropdownMenu | Dialog | ModalDialog | Normal)</value>        
 	      <value>(type=Tooltip | Notification | Utility) &amp; !(name=compiz) &amp; !(title=notify-osd)</value>
 	    </default>
 	  </option>
@@ -169,7 +169,7 @@
 	    <extensible/>
 	    <sort start="2"/>
 	    <default>
-	      <value>animation:Magic Lamp</value>
+	      <value>animation:Zoom</value>
 	    </default>
 	  </option>
 	  <option name="minimize_durations" type="list">
@@ -179,7 +179,7 @@
 	    <min>50</min>
 	    <max>4000</max>
 	    <default>
-	      <value>300</value>
+	      <value>220</value>
 	    </default>
 	  </option>
 	  <option name="minimize_matches" type="list">
@@ -284,7 +284,7 @@
 	    <extensible/>
 	    <sort start="1"/>
 	    <default>
-	      <value>animation:None</value>
+	      <value>animation:Fade</value>
 	    </default>
 	  </option>
 	  <option name="focus_durations" type="list">
@@ -294,7 +294,7 @@
 	    <min>50</min>
 	    <max>4000</max>
 	    <default>
-	      <value>300</value>
+	      <value>150</value>
 	    </default>
 	  </option>
 	  <option name="focus_matches" type="list">
@@ -388,7 +388,7 @@
 	  <option name="glide2_away_position" type="float">
 	    <_short>Away Position</_short>
 	    <_long>Closeness of window to camera at the end of the animation (1.0: Close to camera, -2.0: Away from camera).</_long>
-	    <default>-0.4</default>
+	    <default>-0.1</default>
 	    <min>-2</min>
 	    <max>1</max>
 	    <precision>0.05</precision>
@@ -396,7 +396,7 @@
 	  <option name="glide2_away_angle" type="float">
 	    <_short>Away Angle</_short>
 	    <_long>Angle of window at the end of the animation.</_long>
-	    <default>-45</default>
+	    <default>0</default>
 	    <min>-540</min>
 	    <max>540</max>
 	    <precision>5</precision>
@@ -616,7 +616,7 @@
 	<option name="time_step" type="int">
 	  <_short>Animation Time Step</_short>
 	  <_long>The amount of time in milliseconds between each render of the animation. The higher the number, the jerkier the movements become.</_long>
-	  <default>10</default>
+	  <default>16</default>
 	  <min>1</min>
 	  <max>400</max>
 	</option>
