Description: Set Ubuntu default options
Author: ?

--- a/metadata/expo.xml.in
+++ b/metadata/expo.xml.in
@@ -30,9 +30,6 @@
 				<option name="expo_edge" type="edge">
 					<_short>Expo edge</_short>
 					<_long>Engage wall expo mode edge binding</_long>
-					<default>
-						<edge name="TopLeft"/>
-					</default>
 				</option>
 				<option name="double_click_time" type="int">
 					<_short>Double click time</_short>
@@ -75,7 +72,7 @@
 				<option name="zoom_time" type="float">
 					<_short>Zoom time</_short>
 					<_long>Duration of the zoomout animation</_long>
-					<default>0.5</default>
+					<default>0.3</default>
 					<min>0.1</min>
 					<max>5.0</max>
 					<precision>0.1</precision>
@@ -137,7 +134,7 @@
 				<option name="vp_distance" type="float">
 					<_short>Viewport distance</_short>
 					<_long>Distance between viewports</_long>
-					<default>0.10</default>
+					<default>0.8</default>
 					<min>0.0</min>
 					<max>1.0</max>
 					<precision>0.01</precision>
@@ -188,7 +185,7 @@
 					<option name="vp_brightness" type="float">
 						<_short>Brightness</_short>
 						<_long>Inactive viewport brightness.</_long>
-						<default>75.0</default>
+						<default>80.0</default>
 						<min>0.0</min>
 						<max>100.0</max>
 						<precision>0.1</precision>
--- a/metadata/ezoom.xml.in
+++ b/metadata/ezoom.xml.in
@@ -134,7 +134,6 @@
 		    <option type="key" name="fit_to_window_key">
 			<_short>Fit zoomed area to window</_short>
 			<_long>Zooms in/out so the focused window is zoomed to the maximum while still being fully visible.</_long>
-			<default>&lt;Super&gt;r</default>
 		    </option>
 		</subgroup>
 	    </group>
--- a/metadata/scaleaddon.xml.in
+++ b/metadata/scaleaddon.xml.in
@@ -90,7 +90,7 @@
 			<_long>Font size for window title display</_long>
 			<min>6</min>
 			<max>48</max>
-			<default>10</default>
+			<default>13</default>
 		    </option>
 		    <option name="border_size" type="int">
 			<_short>Title Border Size</_short>
--- a/metadata/snap.xml.in
+++ b/metadata/snap.xml.in
@@ -70,7 +70,8 @@
 		  <max>1</max>
           <default>
             <value>0</value>
-		  </default>
+            <value>1</value>
+	  </default>
 		  <desc>
 			  <value>0</value>
 			  <_name>Screen edges</_name>
@@ -83,7 +84,7 @@
         <option name="resistance_distance" type="int">
           <_short>Edge Resistance Distance</_short>
           <_long>The distance until edge resistance takes place.</_long>
-          <default>30</default>
+          <default>18</default>
           <min>1</min>
           <max>100</max>
         </option>
--- a/metadata/staticswitcher.xml.in
+++ b/metadata/staticswitcher.xml.in
@@ -132,12 +132,12 @@
 		<option name="auto_change_vp" type="bool">
 		    <_short>Auto Change Viewport</_short>
 		    <_long>Change to the viewport of the selected window while switching</_long>
-		    <default>false</default>
+		    <default>true</default>
 		</option>
 		<option name="popup_delay" type="float">
 		    <_short>Popup Window Delay</_short>
 		    <_long>Time (in s) the popup window should be delayed before appearing</_long>
-		    <default>0.0</default>
+		    <default>0.2</default>
 		    <min>0.0</min>
 		    <max>2.0</max>
 		    <precision>0.05</precision>
@@ -145,7 +145,7 @@
 		<option name="mouse_select" type="bool">
 		    <_short>Allow Mouse Selection</_short>
 		    <_long>Allow selection of windows from the switcher window with the mouse</_long>
-		    <default>false</default>
+		    <default>true</default>
 		</option>
 	    </group>
 	    <group>
@@ -153,21 +153,21 @@
 		<option name="saturation" type="int">
 		    <_short>Saturation</_short>
 		    <_long>Amount of saturation in percent</_long>
-		    <default>50</default>
+		    <default>100</default>
 		    <min>0</min>
 		    <max>100</max>
 		</option>
 		<option name="brightness" type="int">
 		    <_short>Brightness</_short>
 		    <_long>Amount of brightness in percent</_long>
-		    <default>50</default>
+		    <default>100</default>
 		    <min>0</min>
 		    <max>100</max>
 		</option>
 		<option name="opacity" type="int">
 		    <_short>Opacity</_short>
 		    <_long>Amount of opacity in percent</_long>
-		    <default>100</default>
+		    <default>0</default>
 		    <min>0</min>
 		    <max>100</max>
 		</option>
--- a/metadata/wall.xml.in
+++ b/metadata/wall.xml.in
@@ -30,7 +30,7 @@
 				<option name="preview_timeout" type="float">
 					<_short>Switch Target Preview Visibility Time</_short>
 					<_long>Duration (in s) the switch target preview should remain visible after switching ends.</_long>
-					<default>0.4</default>
+					<default>0.2</default>
 					<min>0.0</min>
 					<max>2.0</max>
 					<precision>0.05</precision>
@@ -38,7 +38,7 @@
 				<option name="preview_scale" type="int">
 					<_short>Preview Scale</_short>
 					<_long>Size of the preview in %</_long>
-					<default>100</default>
+					<default>130</default>
 					<min>0</min>
 					<max>400</max>
 				</option>
@@ -52,7 +52,7 @@
 				<option name="border_width" type="int">
 					<_short>Border Width</_short>
 					<_long>Width of the border between the previews</_long>
-					<default>10</default>
+					<default>7</default>
 					<min>0</min>
 					<max>30</max>
 				</option>
@@ -60,10 +60,10 @@
 					<_short>Outline Color</_short>
 					<_long>Outline color of the switcher window.</_long>
 					<default>
-						<red>0x3333</red>
-						<green>0x3333</green>
-						<blue>0x3333</blue>
-						<alpha>0xd998</alpha>
+						<red>0xffff</red>
+						<green>0xffff</green>
+						<blue>0xffff</blue>
+						<alpha>0x3232</alpha>
 						</default>
 				</option>
 				<subgroup>
@@ -73,30 +73,30 @@
 						<_short>Base Color</_short>
 						<_long>First color for the background gradient of the switcher window.</_long>
 						<default>
-							<red>0xcccc</red>
-							<green>0xcccc</green>
-							<blue>0xe665</blue>
-							<alpha>0xd998</alpha>
+							<red>0x0000</red>
+							<green>0x0000</green>
+							<blue>0x0000</blue>
+							<alpha>0x6464</alpha>
 						</default>
 					</option>
 					<option name="background_gradient_highlight_color" type="color">
 						<_short>Highlight Color</_short>
 						<_long>Second color for the background gradient of the switcher window.</_long>
 						<default>
-							<red>0xf332</red>
-							<green>0xf332</green>
-							<blue>0xffff</blue>
-							<alpha>0xd998</alpha>
+							<red>0x0000</red>
+							<green>0x0000</green>
+							<blue>0x0000</blue>
+							<alpha>0x6464</alpha>
 						</default>
 					</option>
 					<option name="background_gradient_shadow_color" type="color">
 						<_short>Shadow Color</_short>
 						<_long>Third color for the background gradient of the switcher window.</_long>
 						<default>
-							<red>0xf332</red>
-							<green>0xf332</green>
-							<blue>0xffff</blue>
-							<alpha>0xd998</alpha>
+							<red>0x0000</red>
+							<green>0x0000</green>
+							<blue>0x0000</blue>
+							<alpha>0x6464</alpha>
 						</default>
 					</option>
 				</subgroup>
@@ -107,20 +107,20 @@
 						<_short>Base Color</_short>
 						<_long>First color for the thumb gradient of the switcher window.</_long>
 						<default>
-							<red>0x3333</red>
-							<green>0x3333</green>
-							<blue>0x3333</blue>
-							<alpha>0x5999</alpha>
+							<red>0x5555</red>
+							<green>0x5555</green>
+							<blue>0x5555</blue>
+							<alpha>0x3232</alpha>
 						</default>
 					</option>
 					<option name="thumb_gradient_highlight_color" type="color">
 						<_short>Highlight Color</_short>
 						<_long>Second color for the thumb gradient of the switcher window.</_long>
 						<default>
-							<red>0x3fff</red>
-							<green>0x3fff</green>
-							<blue>0x3fff</blue>
-							<alpha>0x3fff</alpha>
+							<red>0x5555</red>
+							<green>0x5555</green>
+							<blue>0x5555</blue>
+							<alpha>0x3232</alpha>
 						</default>
 					</option>
 				</subgroup>
@@ -134,7 +134,7 @@
 							<red>0xffff</red>
 							<green>0xffff</green>
 							<blue>0xffff</blue>
-							<alpha>0xf332</alpha>
+							<alpha>0xffff</alpha>
 						</default>
 					</option>
 					<option name="thumb_highlight_gradient_shadow_color" type="color">
@@ -144,7 +144,7 @@
 							<red>0xdfff</red>
 							<green>0xdfff</green>
 							<blue>0xdfff</blue>
-							<alpha>0xa665</alpha>
+							<alpha>0xffff</alpha>
 						</default>
 					</option>
 				</subgroup>
@@ -348,7 +348,7 @@
 				<option name="edgeflip_move" type="bool">
 					<_short>Edge Flip Move</_short>
 					<_long>Flip viewport when moving a window to a screen edge</_long>
-					<default>true</default>
+					<default>false</default>
 				</option>
 				<option name="edgeflip_dnd" type="bool">
 					<_short>Edge Flip DnD</_short>
